Formula/v/vipsdisp.rb

```diff
@@ -0,0 +1,33 @@
+class Vipsdisp < Formula
+  desc "Viewer for large images"
+  homepage "https://github.com/jcupitt/vipsdisp"
+  url "https://github.com/jcupitt/vipsdisp/releases/download/v3.1.0/vipsdisp-3.1.0.tar.xz"
+  sha256 "5c40e71c9c60232dcbf2e1c389295a4a102a27603bce994dbb2e35ff4f1844db"
+  license "MIT"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "gtk4"
```

Comment:
```suggestion
  depends_on "glib"
  depends_on "gtk4"
```

Reply:
I'm happy to make the change of course, but gtk4 depends on glib, so is this necessary?

Reply:
yeah, we are transitively declaring all the deps

Reply:
you can see in here, https://github.com/Homebrew/homebrew-core/pulls?q=is%3Apr+declare+indirect+deps+with+linkage

Reply:
I'm not sure this is a good idea, there are a lot more transitive deps, perhaps more than 40. Should I list all of them?

Reply:
it is just one dependency linked in the runtime, that is why only glib got declared